apps/gallery/js/thumbnail_item.js

```diff
@@ -39,7 +39,9 @@ ThumbnailItem.formatter = new navigator.mozL10n.DateTimeFormat();
 
 ThumbnailItem.prototype.localize = function() {
   var date = new Date(this.data.date);
-  var imageCreatedAt = navigator.mozL10n.get('image-created-at');
-  var label = ThumbnailItem.formatter.localeFormat(date, imageCreatedAt);
+  var description = navigator.mozL10n.get(
+    !this.data.metadata.video ?
+    'imageDescriptionShort' : 'videoDescriptionShort');
```

Comment:
nit: Sorry for missing this before, for readability we should put !this.data.metadata.video ? on the same line as the open parenthesis like
  var description = navigator.mozL10n.get(!this.data.metadata.video ?
    'imageDescriptionShort' : 'videoDescriptionShort');